### ISSUE TITLE
Empty queue illustration

### ISSUE THREAD

**Issue by richardolsson** [zetkin/call.zetk.in#152]
Add an illustration to the "queue empty" page.

**Comment by richardolsson** [comment]
Screenshot for context.

![2_queue-empty](https://user-images.githubusercontent.com/550212/31385523-d9f99b2c-adc3-11e7-8764-bb428cf08605.png)

**Comment by richardolsson** [comment]
Looking great! 🎆 

Will move ahead and implement these!

**Comment by rebeccatencic** [comment]
The illustrations above should be moved to the "Call reported" issue, right? The empty queue suggestions are the ones that look like this at the moment. Which one do we move ahead with?
![skarmavbild 2017-12-21 kl 12 26 35](https://user-images.githubusercontent.com/26344452/34254193-43712ff6-e64b-11e7-96d2-0f1cb14d1d69.png)
![skarmavbild 2017-12-21 kl 12 28 50](https://user-images.githubusercontent.com/26344452/34254194-438c7f86-e64b-11e7-9fa1-749fa28d787d.png)

**Comment by richardolsson** [comment]
That's right! I've deleted the comments containing the unrelated illustrations (they have already been implemented in #155).

Both of these look great! I personally prefer the second, close-up one. But I think it might look even better with the perspective rotated for the desk to be perfectly horizontal, and maybe so that the desk is cropped by the circle instead of showing the black frame of the desk?

**Comment by richardolsson** [comment]
P.S. I love how the illustration is repeated on the screen. 😂
